cluster-autoscaler/cloudprovider/oci/nodepools/oci_manager_test.go

```diff
@@ -384,3 +388,70 @@ func TestRemoveInstance(t *testing.T) {
 		}
 	}
 }
+
+func TestNodeGroupFromArg(t *testing.T) {
+	var nodeGroupArg = "clusterId:testClusterId,compartmentId:testCompartmentId,nodepoolTags:ca-managed=true&namespace.foo=bar,min:1,max:5"
```

Comment:
Minor nit, but can we update the IDs to look like real ocids just in case there are weird parsing bugs. For example

```
ocid1.cluster.oc1.test-region.test
ocid1.compartment.oc1.test-region.test
```

Reply:
done.